main.go

```diff
@@ -47,6 +50,7 @@ func parseFlags(progname string, args []string) (*Config, string, error) {
 	flags.BoolVar(&cfg.excludeMinor, "no-minor", false, "exclude pre-release version (default: false)")
 	flags.BoolVar(&cfg.excludePrefix, "no-prefix", false, "exclude version prefix (default: false)")
 	flags.BoolVar(&cfg.guardRelease, "guard", false, "ignore shorthand options if version contains pre-release (default: false)")
+	flags.Var(&cfg.targetRelease, "target-release", "set which version component (major, minor or patch) will be bumped if version contains pre-release (default: patch)")
```

Comment:
What do you think about naming this option `-bump-to-next`? So that the invocation looks like
```sh
$ git-semver -bump-to-next minor
```
This would read intuitively.

Reply:
Hi, sorry for the long response time. I am not a native english speaker so I cannot tell what text reads most intuitively. Therefore, I have no issue with your suggestion of using -bump-to-next.